tests/CMakeLists.txt

```diff
@@ -55,6 +55,6 @@ else()
 
 endif()
 
-target_link_libraries(behaviortree_cpp_test ${BTCPP_LIBRARY} bt_sample_nodes foonathan::lexy)
-target_include_directories(behaviortree_cpp_test PRIVATE include ${PROJECT_SOURCE_DIR}/3rdparty)
```

Comment:
Exposing the whole 3rdparty folder as a include dir is a bad pattern. We should only link in what we need